client/table.go

```diff
@@ -211,9 +211,16 @@ func (m *model) encodeColumnKey(primaryKey []byte, colID uint32) []byte {
 // CreateTable creates a table from the specified schema. Table creation will
 // fail if the table name is already in use.
 func (db *DB) CreateTable(schema proto.TableSchema) error {
+	_, err := db.CreateTableInner(schema)
+	return err
+}
+
+// CreateTableInner is the implementation of CreateTable. It is exposed so it
+// can be used on the server as well.
+func (db *DB) CreateTableInner(schema proto.TableSchema) (*proto.TableDescriptor, error) {
```

Comment:
The current convention for "internal" methods would be to name this `InternalCreateTable`.